src/main/java/wolox/training/models/Book.java

```diff
@@ -45,6 +47,9 @@
     @Column(nullable = false, unique = true)
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users;
+
```

Comment:
Add a get with Collections.unmodifiableList for users list.

Reply:
Add Collections.unmodifiableList in the get of users list.